src/continuation/continuation_stability_measures.jl

```diff
@@ -67,43 +53,61 @@ function global_continuation(
 
     rmaps = match_sequentially!(attractors_cont, acam.matcher; pcurve, ds = referenced_dynamical_system(mapper))
 
-    # This is the second difference: we change the output to something more pleasant to handle
-    transposed_measures_cont = Dict{String, Vector{Dict{Int64, Float64}}}()
+    # This is the third difference in global continuation
+    for i in 2:length(pcurve)
+        for dict in values(measures_cont[i])
+            swap_dict_keys!(dict, rmaps[i-1])
+        end
+    end
+    transposed = Dict{String, Vector{Dict{Int64, Float64}}}()
     for measure in measures_cont[1]
         measure_name = measure[1]
-        transposed_measures_cont[measure_name] = Vector{Dict{Int64, Float64}}()
+        transposed[measure_name] = Vector{Dict{Int64, Float64}}()
     end
+
     for measures in measures_cont
         for (measure_name, measure_dict) in measures
-            push!(transposed_measures_cont[measure_name], measure_dict)
+            push!(transposed[measure_name], measure_dict)
         end
     end
-
-    # And lastly, we also match sequentially all dictionaries here
-    for measure_cont in values(measures_cont)
-        match_sequentially!(measure_cont, rmaps)
-    end
+    measures_cont = transposed
```

Comment:
unecessary. Just retun `transposed`. You are changing the type of `measures_cont` which is in general bad practice in Julia.